src/rime/gear/charset_filter.cc

```diff
@@ -24,6 +24,7 @@ bool is_extended_cjk(uint32_t ch)
       (ch >= 0x2B740 && ch <= 0x2B81F) ||  // CJK Unified Ideographs Extension D
       (ch >= 0x2B820 && ch <= 0x2CEAF) ||  // CJK Unified Ideographs Extension E
       (ch >= 0x2CEB0 && ch <= 0x2EBEF) ||  // CJK Unified Ideographs Extension F
+      (ch >= 0x30000 && ch <= 0x3134A) ||  // CJK Unified Ideographs Extension G
```

Comment:
Please edit the range to match the standard:
https://en.wikipedia.org/wiki/CJK_Unified_Ideographs_Extension_G

    Range | U+30000..U+3134F

Reply:
#393 edited